taca/analysis/analysis.py

```diff
@@ -1,19 +1,20 @@
 """ Analysis methods for TACA """
 import csv
+from datetime import datetime
```

Comment:
Oh, but I think I understand :-)

Sorry I didn't plan to change so much apart from the logging stuff... My thinking is that it's simplest to group all standard lib imports together, then non-standard, and lastly intra-package imports.

If you think it's clearer to separate standard lib imports because some are "from this import that" than I will correct it!